README.md

```diff
@@ -4,6 +4,8 @@ A simple token management tool.
 
 ## Usage
 
+Note that for all commands below you will need to replace `toker` with `go run main.go` unless you've compiled the code into a binary named `toker`. Before running, you may need to run `rm go.sum;go get -u ./...`
```

Comment:
`Before running, you may need to run `rm go.sum;go get -u ./...``

This shouldn't be true, I'm not sure why you ran into those issues.  You could enumerate the steps though, which would be (at a high level):

```
git clone ....
cd toker
go get ./...
alias toker="go run main.go"
toker help
```